lighthouse-cli/bin.ts

```diff
@@ -143,6 +147,10 @@ if (!cliFlags.outputPath && cliFlags['output-path']) {
   cliFlags.outputPath = cliFlags['output-path'];
 }
 
+if (!cliFlags.maxWaitForLoad && cliFlags['max-wait-for-load']) {
```

Comment:
suspect that's it. Just complaining out loud that that restriction continues to hold us back.